README.md

```diff
@@ -15,7 +15,7 @@ connected adaptive quadtrees or octrees in parallel.
 ## Installation
 If you have not yet installed Julia, please [follow the instructions for your
 operating system](https://julialang.org/downloads/platform/). P4est.jl works
-with Julia v1.5.
+with Julia v1.6.
```

Comment:
If we require this, we should also change `Project.toml` accordingly. If not, we should probably also add tests on Julia v1.5.

Reply:
Good point. Fixed in 1b9a2b1.